tests/test_api_v1.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import unittest
+
+from purview.api.v1 import app
+
+
+class TestAPI(unittest.TestCase):
+
+    def setUp(self):
+        self.app = app.test_client()
+
+    def tearDown(self):
+        pass
+
+    def test_about_endpoint(self):
+        rv = self.app.get('/about')
+        assert b'Version:' in rv.data
```

Comment:
Instead of using bytes, you should probably decode `rv.data` instead.

Reply:
I saw pytest docs using bytes, hence I used it.

Reply:
Sure. I will change it